service/deployer/deployer.go

```diff
@@ -24,30 +25,32 @@ type DeployerType string
 // Config represents the configuration used to create a Deployer.
 type Config struct {
 	// Dependencies.
+	FileSystem       afero.Fs
```

Comment:
The injection of the dependencies is a bit tedious because the service structure is very nested. I think in the future we can flatten this a bit.